portfolio/src/main/webapp/background.html

```diff
@@ -5,26 +5,28 @@
     <title>Hadley Shapland</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+
+    <!-- include jquery -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+    <script>
+      $(function(){
```

Comment:
Could you move this logic to script.js to try to keep the Javascript and HTML separate? This should also help with reducing duplicated logic.

Reply:
Good point, just fixed it!